src/workloads/query/TelemetryWithRedaction.yml

```diff
@@ -1,10 +1,10 @@
 SchemaVersion: 2018-07-01
```

Comment:
I left the file name as is so it would be easier to see the diff, but I can rename the file to QueryStatsWithRedaction? Although redaction is probably not the right word now either?

Reply:
Ha! True. Our proposal now is either "OneWayTokenization" or "apply HMAC" or "apply HMAC hashing", although this workload does it both with and without that on, right? So I think maybe remove it from the filename, but in the actors you can say `FindConcurrentlyWithHMACQueryStats` and `FindConcurrentlyWithQueryStatsWithoutHMAC`, etc.?